manifests/n/Nvidia/ProfileInspector/2.4.0.4/Nvidia.ProfileInspector.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with komac v2.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Nvidia.ProfileInspector
+PackageVersion: 2.4.0.4
+PackageLocale: en-US
+Publisher: Orbmu2k
+PublisherUrl: https://github.com/Orbmu2k
+Author: Orbmu2k
+PackageName: Nvidia Profile Inspector
+License: MIT
+LicenseUrl: https://github.com/Orbmu2k/nvidiaProfileInspector/blob/master/LICENSE
+Copyright: Copyright (c) 2016 Orbmu2k

```

Comment:
This matches the license page, but not what is shown in the application
![image](https://github.com/user-attachments/assets/c5da37e4-3beb-412e-8928-0a3487feb3e8)

Reply:
Hmm, not quite sure how to resolve this since the LICENSE from the repo hasn't been modified for more than 8 years now.